hound_test.rb

```diff
@@ -0,0 +1,10 @@
+puts 'single quotes'
+puts "double quotes"
+
+puts "this is #{'interpolation'}"
+
+puts "this " + "is " + "an " + "concatenation"
+puts "this " "also " "is " "an " "concatenation"
+
+puts 'this ' + 'is ' + 'an ' + 'concatenation'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.